.github/workflows/node-ci.yml

```diff
@@ -226,11 +242,15 @@ jobs:
                   git checkout $REF
 
             - name: Install & cache node_modules
+              # Only bother if there are no releases in progress
+              if: steps.check-release.outputs.release-count == '0'
```

Comment:
**note:** The job failure should mean this doesn't run, but no harm in being explicit.

Reply:
Yeah, that's reasonable. I added these before I changed to failing hard and I just didn't want to do the work of deleting it. Totally lazy. I'll just delete these. :grin: